src/components/Video/index.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react'
+
+export default class Video extends Component {
+   static propTypes = {
+      className: React.PropTypes.string,
+      src: React.PropTypes.string
+   }
+
+   constructor(props, context) {
+      super(props, context)
+
+      this.state = {
+         size: this.getWindowSize()
+      }
+
+      this.windowSizeUpdated = this.windowSizeUpdated.bind(this)
+   }
+
+   componentDidMount() {
+      this.setVideoContainer()
+      window.addEventListener('resize', this.windowSizeUpdated, false)
+   }
+
+   componentDidUpdate() {
+      this.setVideoContainer()
+   }
+
+   componentWillUnmount() {
+      window.removeEventListener('resize', this.windowSizeUpdated)
+   }
+
+   getAttributeForCurrentSize(attributeValue) {
+      const currentSize = this.state.size
+      const fragments = attributeValue.split(' ')
+
+      for (const fragment of fragments) {
+         const charSet = fragment.match(/\[([abcdef,-]+)\]/i)
+         if (Array.isArray(charSet) && charSet.length === 2) {
+            const charRegexp = new RegExp(`[${charSet[1]}]`, 'i')
+            const match = currentSize.match(charRegexp)
+            if (Array.isArray(match)) {
+               return fragment.replace(charSet[0], '')
+            }
+         } else {
+            return fragment
+         }
+      }
+
+      return null
+   }
+
+   getWindowSize() {
+      const windowWidth = document.documentElement.clientWidth
+
+      if (windowWidth >= 1650) {
+         return 'e'
+      } else if (windowWidth >= 1300) {
+         return 'd'
+      } else if (windowWidth >= 992) {
+         return 'c'
+      } else if (windowWidth >= 768) {
+         return 'b'
+      } else if (windowWidth >= 0) {
+         return 'a'
+      }
+
+      return null
+   }
+
+   setVideoContainer() {
```

Comment:
I will be using this function as soon as it displays the video correctly. This function is for sizing the video and positioning it correctly.